models/repo_collaboration.go

```diff
@@ -183,3 +183,29 @@ func (repo *Repository) DeleteCollaboration(uid int64) (err error) {
 
 	return sess.Commit()
 }
+
+func (repo *Repository) getRepoTeams(e Engine) (teams []*Team, err error) {
+	return teams, e.
+		Join("INNER", "team_repo", "team_repo.team_id = team.id").
+		Where("team.org_id = ?", repo.OwnerID).
+		And("team_repo.repo_id=?", repo.ID).
+		OrderBy("CASE WHEN name LIKE '" + ownerTeamName + "' THEN '' ELSE name END").
+		Find(&teams)
+}
+
+func (repo *Repository) GetRepoTeams() ([]*Team, error) {
```

Comment:
Please add a comment in the form:
```
// GetRepoTeams .... (description)
func (repo *Repository) GetRepoTeams() ([]*Team, error) {
```
in order to pass the CI checks.

Reply:
Done